avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -1011,16 +1047,33 @@ def updatePositionC(cfg, particles, dem, force, fields, int typeStop=0):
 
   if stop:
     particles['iterate'] = False
+    particles['stoppedParticles'] = particles
+    massStopped = massStopped + massFlowing
+
     if typeStop == 1:
       log.debug('stopping initial particle distribution')
     else:
       log.debug('stopping because of %s stopCriterion.' % (cfg['stopCritType']))
 
+  particles['massStopped'] = - massStopped
```

Comment:
Do we need the negativ here for computations further down the line? For me, stopped mass has to be positive? As a negative stopping would imply acceleration in my opinion